app/src/main/java/nl/tudelft/b_b_w/view/ContactAdapter.java

```diff
@@ -135,12 +135,12 @@ public View getView(final int position, View convertView, ViewGroup parent) {
             view = inflater.inflate(R.layout.simple_list_item_1, null);
         }
         TextView nameItemText = (TextView)view.findViewById(R.id.list_item_name);
-        nameItemText.setText(blcController.getContactName(blcController.getBlocks(ownerName).get(position).getOwnHash()));
+        nameItemText.setText(blockController.getContactName(blockController.getBlocks(ownerName).get(position).getOwnHash()));
         TextView ibanItemText = (TextView)view.findViewById(R.id.list_item_iban);
-        ibanItemText.setText(blcController.getBlocks(ownerName).get(position).getIban());
+        ibanItemText.setText(blockController.getBlocks(ownerName).get(position).getIban());
         ImageView pic = (ImageView)view.findViewById(R.id.trust_image);
         pic.setImageResource(
-                getImageNo(blcController.getBlocks(ownerName).get(position).getTrustValue()));
+                getImageNo(blockController.getBlocks(ownerName).get(position).getTrustValue()));
         Button revokeButton = (Button)view.findViewById(R.id.revoke_btn);
```

Comment:
This is debatable. Since revoke_btn isn't a variable. This is only an ID, and I doubt that this affects readability.